cybergis/general/comet.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+
+from .base import SBatchScript
+from .job import SlurmJob
+from .connection import SSHConnection
+
+
+logger = logging.getLogger("cybergis")
+
+
+class CometSBatchScript(SBatchScript):
+    pass
+
+
+class CometJob(SlurmJob):
+
+    JOB_ID_PREFIX = "Comet_"
```

Comment:
@CarnivalBug How much difference does this class introduce in compared with the SlurmJob class?